feat: add setter and getter apis to specify udp port range for webrtc (#39046)
electron/electron: docs/api/web-contents.md

exposed via WebRTC. See [BrowserLeaks](https://browserleaks.com/webrtc) for
more details.

#### `contents.getWebRTCUDPPortRangePolicy()`

Returns `Object`:

* `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.

> * `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.

Reviewer: Poking around in Chromium internals, it sounds like these would default to `{ min: 0, max: 0 }` which would apply no restriction on port range. Should we include a note (and maybe a unit test) about the default value/behavior of `setWebRTCUDPPortRangePolicy` hasn't been called?

Author: Yeah, a note for the default behavior is reasonable

#### `contents.setWebRTCUDPPortRangePolicy(udpPortRange)`

* `udpPortRange` Object
  * `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
  * `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.

> Setting the WebRTC UDP Port Range policy allows you to restrict the udp port range assigned to apps via WebRTC.

Reviewer: If someone sets a policy and then wants to remove it, would they pass `{ min: 0, max: 0 }`? Is it worth including that in the docs?

Author: Sure, this is a reasonable scene, thanks for your review.

exposed via WebRTC. See [BrowserLeaks](https://browserleaks.com/webrtc) for
more details.

> #### `contents.getWebRTCUDPPortRangePolicy()`

Reviewer 2: ```suggestion
#### `contents.getWebRTCUDPPortRange()`
```

Reviewer 3: Thanks for your review:). 
I thought the udp port range config corresponds to the chrome://policy entry, and reference to another WebRtc api setWebRTCIPHandlingPolicy, I added the word "policy " in the API. 
But it is a bit long and tedious for the api. I will follow your suggestion to simplify it.

Author: Thanks for your review:).
I thought the udp port range config corresponds to the chrome://policy entry, and reference to another WebRtc api setWebRTCIPHandlingPolicy, I added the word "policy " in the API.
But it is a bit long and tedious for the api. I will follow your suggestion to simplify it.

Returns `Object`:

> * `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
> * `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.

Reviewer 2: ```suggestion
* `min` Integer - The minimum UDP port number that WebRTC should use.
* `max` Integer - The maximum UDP port number that WebRTC should use.
```

> Note that as poking round chromium internal , it would default to { min: 0, max: 0 } , which would apply no restriction on udp port range.

Reviewer 2: ```suggestion
By default this value is `{ min: 0, max: 0 }` , which would apply no restriction on the udp port range.
```

> #### `contents.setWebRTCUDPPortRangePolicy(udpPortRange)`

Reviewer 2: ```suggestion
#### `contents.setWebRTCUDPPortRange(udpPortRange)`
```

* `udpPortRange` Object

>   * `min` Integer - The minimum port number of WebRTC UDP Port Range Policy.
>   * `max` Integer - The maximum port number of WebRTC UDP Port Range Policy.

Reviewer 2: ```suggestion
  * `min` Integer - The minimum UDP port number that WebRTC should use.
  * `max` Integer - The maximum UDP port number that WebRTC should use.
```

> Setting the WebRTC UDP Port Range policy allows you to restrict the udp port range assigned to apps via WebRTC. Note that when setting to { min: 0, max: 0 }，no restriction is applied on udp port range.

Reviewer 2: ```suggestion
Setting the WebRTC UDP Port Range allows you to restrict the udp port range used by WebRTC.  By default the port range is unrestricted.  

*Note:** To reset to an unrestricted port range this value should be set to `{ min: 0, max: 0 }`.
```
